src/main/java/io/gravitee/common/utils/RxHelper.java

```diff
@@ -116,6 +123,27 @@ public static <R> FlowableTransformer<R, R> retryFlowable(
             );
     }
 
+    public static <R> FlowableTransformer<R, R> retryExponentialBackoffFlowable(final long firstDelay, final TimeUnit unit) {
```

Comment:
Can you add a piece of javadoc to explain each parameter and the expected behavior 🙏 (ex: default maxDaly, how the factor influences the delay for each retry, ...)

Reply:
of course sir!